### ISSUE TITLE
Battery Status: Change Output when Charging

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#10148]
### Describe the solution you'd like

Currently, when reading the battery status when a device is charging NVDA will read "battery percentage AC power on". For the average user, the use of "AC power on" may not make any sense; could we consider changing the use of "AC power on" to "charging" instead?

**Comment by lukaszgo1** [comment]
This message is at the moment the same  no matter if battery is charging or charged, so AC power on is technically more correct.

**Comment by XLTechie** [comment]
ukasz Golonka wrote:

> This message is at the moment the same no matter if battery is charging or charged, so AC power on is technically more correct.

Well if we're being technical, it should be DC power on.:)

Perhaps "Plugged in" would be more accurate for the generic case.

**Comment by elliott94** [comment]
If the battery level is at 100%, could we not instead then change this to "charged"?

**Comment by Adriani90** [comment]
actually, when 100% has been reached, the batery is charging and the 100% status is maintained while the cable is plugged in. So actually the batery gets power all the time even thou 100% has been reached.

**Comment by Adriani90** [comment]
This is now changed as expected in NVDA 2023.1 RC1. NVDA reports "charging", when the cable is pluged in. How ever, this occurs in Windows 11, I cannot speak for Windows 10. @lukaszgo1 is this also the case in Windows 10?

**Comment by Adriani90** [comment]
It seems this has been changed in NVDA also for Windows 10. I am closing this in favor of #14726 since there are arguments that suggest a more exact approach than just reporting "charging".
